components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultAuthenticationRequestHandler.java

```diff
@@ -82,6 +82,7 @@
     private static final Log log = LogFactory.getLog(DefaultAuthenticationRequestHandler.class);
     private static final Log AUDIT_LOG = CarbonConstants.AUDIT_LOG;
     private static volatile DefaultAuthenticationRequestHandler instance;
+    private static String EXTEND_REMEMBER_ME_SESSION_ON_AUTH = "TimeConfig.ExtendRememberMeSessionTimeoutOnAuth";
```

Comment:
make final

Reply:
We can move this to FrameworkConstants as well.
https://github.com/wso2/carbon-identity-framework/blob/master/components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java

Reply:
Fixed: 2dc93b4f4987b4ac95f607c667d4ec86c75ec661